### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by amitAMPL**
new project structure.

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 const routes: Routes = [
-  { path:'', redirectTo:'tasks', pathMatch:'full' },
-  { path:'tasks',component:TasksComponent }
+  {
+    path: '', redirectTo: 'list', pathMatch: 'full'
+  },
+  // {
+  //   path: 'tasks', loadChildren: () => import('./list/tasks/tasks.module').then(m => m.TasksModule)
+  // },
```
Comment (prodeep-acme) [inline src/app/app-routing.module.ts]: Remove this commented code

File: src/app/list/tasks/tasks.component.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+// import { SidebarComponent } from '../sidebar/sidebar.component';
```
Comment (prodeep-acme) [inline src/app/list/tasks/tasks.component.ts]: delete this commented code

File: src/app/list/tasks/tasks.module.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+// import { FormsModule } from '@angular/forms';
+// import { MatCardModule } from '@angular/material/card';
+// import { MatGridListModule } from '@angular/material/grid-list';
+// import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+// import { MatAutocompleteModule } from '@angular/material/autocomplete';
+// import { MatSliderModule } from '@angular/material/slider';
+// import { MatSelectModule } from '@angular/material/select';
+// import { MatRadioModule } from '@angular/material/radio';
+// import { MatInputModule } from '@angular/material/input';
+// import { MatFormFieldModule } from '@angular/material/form-field';
```
Comment (prodeep-acme) [inline src/app/list/tasks/tasks.module.ts]: delete all these comments

File: src/app/shared/shared.module.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import { CommonModule } from '@angular/common';
+import { SidebarComponent } from '../list/sidebar/sidebar.component';
+import { HeaderComponent } from '../list/header/header.component';
+
+
+
+@NgModule({
+  declarations: [
+    // SidebarComponent,
+    // HeaderComponent
```
Comment (prodeep-acme) [inline src/app/shared/shared.module.ts]: As of now, you have no requirement for a shared module. please delete it.

**Review by prodeep-acme — COMMENTED** [review]
inside content html, don't use the tag of task component. use router-outlet instead.

File: src/app/list/sidebar/sidebar.component.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      </button>
+      <mat-divider></mat-divider>
+      <div class="addTaskSection"><button class="plusIcon">+</button> New list</div>
+    </mat-sidenav>
+
+    <!-- <mat-sidenav-content>
+      <div class="content">
+        <app-content></app-content>
+      </div>
+    </mat-sidenav-content> -->
```
Comment (prodeep-acme) [inline src/app/list/sidebar/sidebar.component.html]: remove commented code

File: src/app/list/list.module.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+
+@NgModule({
+  declarations: [
+    ListComponent,
+    HeaderComponent,
+    SidebarComponent,
+    TaskComponent,
+    AllTaskListComponent,
+    AddTaskComponent
```
Comment (prodeep-acme) [inline src/app/list/list.module.ts]: remove the unused component from the list module. declare them in content.module instead.

File: src/app/list/list.module.ts [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    MatSliderModule,
+    MatAutocompleteModule,
+    NgbModule,
+    MatGridListModule,
+    MatCardModule,
+    FormsModule,
+    MatDividerModule,
+    MatListModule,
+    MatExpansionModule,
+    MatCheckboxModule,
```
Comment (prodeep-acme) [inline src/app/list/list.module.ts]: Also remove unused modules from here

File: src/app/list/list-routing.module.ts [hunk 8]
```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { ListComponent } from './list.component';
+
+const routes: Routes = [
+    {
+      path:'', component: ListComponent, children: [
```
Comment (prodeep-acme) [inline src/app/list/list-routing.module.ts]: start children array from next line

**Review by prodeep-c — APPROVED** [review]

**Pull request merged**